src/canvas/Renderer.tsx

```diff
@@ -104,14 +140,20 @@ class Renderer implements IRenderer {
 
     private _drawPatternPaths = (): void => {
         const context = this._context;
-        context.lineWidth = 3;
+        context.lineWidth = 5;
         context.lineJoin = 'round';
         context.lineCap = 'round';
         
         const paths = this._document.getPatternPaths();
         paths.forEach(path => {
             const path2D = path.getPath2D();
-            const pathColor = PatternPathColor.get(path.getType());
+            let pathColor: string | undefined;
+            if (path.isSelected() || path.isHighlighted()){
+                pathColor = '#000000';
```

Comment:
We can add PatternPathColor.set(PatternPathType.Selected, '#0000000'); to PatternPathColor and use instead.

Reply:
I thought about that that but it seemed weird since a PatternPathType is supposed to refer to weather a path is a seam, fold or edge, which has nothing to do with weather a path is selected or not.

Reply:
I agree it shouldnt be set manually like it is now, though. I just havent found a way I like to do it.

Reply:
We changed the map to accept PatterPathType or string as keys in order to specify a selected color without having to create a new PatternPathType.